geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/command/ExecuteRegionFunction66.java

```diff
@@ -321,24 +321,29 @@ public void cmdExecute(final Message clientMessage, final ServerConnection serve
 
         resultSender.setException(fe);
       } else {
-        logger.warn(LocalizedMessage.create(
-            LocalizedStrings.ExecuteRegionFunction_EXCEPTION_ON_SERVER_WHILE_EXECUTIONG_FUNCTION_0,
-            function), fe);
-        sendException(hasResult, clientMessage, message, serverConnection, fe);
+        if (setLastResultReceivedElseReturnFalseIfAlreadySet(resultSender)) {
```

Comment:
I was thinking just setLastResultsRecieved(resultSender) and the return value would be true/false.  False if already set.  Kind of like how in the Set interface, it's not called addAndReturnFalseIfNotAdded() :-)